spec/lucky/router_spec.cr

```diff
@@ -1,7 +1,13 @@
 require "../../spec_helper"
 
 describe Lucky::Router do
+  # TODO: use this once there's a way to scope to just this block
+  # Spec.before_each do
+  #  Lucky::Router.reset!
+  # end
```

Comment:
I'm not sure how to do this, but maybe asking in the forum on Gitter would help. Would definitely be helpful to be able to use this just in a single describe block

Reply:
I asked, you can't. It doesn't matter where the `before_each` is at, it'll apply to ALL `it` blocks. Adding it in causes some other specs to fail, so I'm not sold on it being in there.

Reply:
Hmm. I think that should be something added to Crystal, but yeah I think we can remove the comments and add it where it is necessary like you did in this spec 👍